index.html

```diff
@@ -80,35 +120,50 @@
 	  	<div class="head-body">
 	  		I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
 	  	</div>
-	  	<div>
+	  	<div class="image1 desktop-hide">
 	  		<img src="assets/images/social-media.png" alt="social media">
 	  	</div>
-	  	
+	  </div>
+	  <div class="col-6 hide">
+		<img class="heading-image" width="70%" src="assets/images/heading-desktop.png" alt="social media">
 	  </div>  		
-	  	
-	</div>
+	</div>	
 </section>
 
-<section id="body">
+<section id="body" class="page-body">
 	<div class="grid-body">
-		<h1 class="recent-work">
-			My Recent Works
-		</h1>
+		<div class="row row-margin">
+			<div class="col-4">
+				<h1 class="recent-work">
+					My Recent Works
+				</h1>
+			</div>
+			<div class="col-8 line">
+				<hr class="line2">
+			</div>
+		</div>
 		<div class="stories">
-			<img src="assets/images/placeholder-1.png" alt="stories">
-			<h1 class="work-sample-2"> Multy-Post Stories </h1>
-			<div class="work-sample-detail">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+			<div class="col-7">
+				<img class="desktop-hide" src="assets/images/placeholder-1.png" alt="stories">
+				<img width="90%" class="hide center" src="assets/images/placeholder-2.png" alt="stories">
+			</div>
+			
+			<div class="col-5">
+				<h1 class="work-sample-2"> Multy-Post Stories </h1>
+				<div class="work-sample-detail">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+				</div>
+				<ul class="tags-2">
+					<li class="tag">css</li>
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+				</ul>
+				<a class="see-2" href="#"> See Project</a>
```

Comment:
- [ ] This project section layout is different than the original design. Please fix it. (mobile version)
![image](https://user-images.githubusercontent.com/38635158/173106956-d76f3ca4-4e64-4e74-8f11-08e915409249.png)

- [ ] See project button's text should center. 

![image](https://user-images.githubusercontent.com/38635158/173107261-2523c086-0cd5-4a5b-89a2-18fdc7dda87e.png)

- [ ] Also please apply all buttons hover, active disable style. Some of them have not.


![image](https://user-images.githubusercontent.com/38635158/173107513-15db6cf8-9a81-4acc-825c-0dfdd8bc20e3.png)